specification/hugr.md

```diff
@@ -408,6 +408,9 @@ Some normalizations are possible:
   - If the entry node has only one successor and that successor is the
     exit node, the CFG node itself can be removed.
 
+A `DFB` that simply wires its inputs to its outputs, having no effect on the
```

Comment:
Is this a new node type? I think it's just a special case of an existing one, so might be worth clarifying that.
* If it is indeed a special case, then it'll still have Input and Output children with dataflow edges between those
* But if it's a special kind of node then maybe it has no children. This would be more like Identity, and that'd be nice for uniformity wrt (Remove/Insert)Identity. Note this would be a lot like an Exit node, so maybe we can even *remove* the Exit node type by requiring that the second child of a CFG node be a passthrough node (even if not necessarily the only one)...???

Reply:
So the intention was not to define a new node type here, but maybe there is an argument for doing so, especially if it allows us to remove the `Exit` type. Hmm, that seems like a big change though.

Reply:
Conversely, maybe `identity<T>` is not required. It could just be a `DFG` node with, much like a "passthrough" DFB, all out-ports of the Input node wired to all in-ports of the Output node....

Perhaps the thing to do is to break the passthrough/identity/etc. change out (if it's gonna be big one way or another) and then everything else here seems uncontroversial...

Reply:
https://github.com/CQCL/hugr/issues/763